server/backend/housekeeping/housekeeping.go

```diff
@@ -127,9 +108,11 @@ func (h *Housekeeping) Stop() error {
 
 // run is the housekeeping loop.
 func (h *Housekeeping) run() {
+	housekeepingLastProjectID := database.DefaultProjectID
```

Comment:
It depends. If it needs to share the variable across different parts of an application, it would be better to put it with a mutex in a struct. Also, if we need to share the variable across all instances in a clustered environment, a good approach is to use a distributed key-value store. Using a variable within the function's scope is like it doesn't share the variable across an application.

What do you think is the best in the situation here?

Reply:
Then I think this variable is best fit in here, since this variable only effects  housekeeping's internal `deactivateCandidates` function and corresponding database `FindDeactivateCandidates` function.

Thank you for sharing this!

Reply:
I believe that when two methods share a variable, they become dependent on each other, so it would be good to avoid such dependencies whenever possible. One alternative approach could be passing the variable as a value and updating it directly from the caller's side. What do you think about this idea?

Reply:
Could you see the code changes in the last commit? I am not sure I understand you correctly.

Reply:
@tedkimdev I think your implementation is correct.
Can you review this? @hackerwins